src/main/java/dev/snowdrop/jira/atlassian/model/Release.java

```diff
@@ -1,50 +1,28 @@
 package dev.snowdrop.jira.atlassian.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.util.List;
 
 public class Release {
-	private String jiraProject;
-	private String jiraKey;
-	private String version;
-	private String longVersionName;
-	private String fixVersion;
-	private String EOL;
-	private String releaseDate;
-	private String dueDate;
-	private String dueDateFormatted;
```

Comment:
Such fields are needed to fill the mustache template with dueDate, EOL & our release date